WcaOnRails/db/seeds/teams.seeds.rb

```diff
@@ -0,0 +1,9 @@
+if Team.count == 0
```

Comment:
It's the thing I really wasn't sure about because of the mentioned migration file that should fill the table, but probably there's no reason to do so. I will improve the code now.